docs/src/piccolo/getting_started/setup_mysql.rst

```diff
@@ -0,0 +1,28 @@
+.. _setting_up_mysql:
+
+###########
+Setup MySQL
+###########
+
+Installation
+************
+
+Follow the  `instructions for your OS <https://dev.mysql.com/doc/refman/8.4/en/installing.html>`_.
+
+
+Creating a database
+*******************
+
+Using ``mysql``:
+
+.. code-block:: bash
+
+    mysql -u root -p
+
+Enter the your password and create database:
+
+.. code-block:: bash
+
+    CREATE DATABASE "my_database_name";
+
+For Windows you can use some GUI tool.
```

Comment:
This is also true of MacOS etc, so maybe change it to:

`Alternatively, use a GUI tool.`

Reply:
Thanks.  I would change that.